### PR TITLE
bin: rook moved config values and exoscale init changes

### LINKED ISSUE #1597: Network policies aren't configred properly for exoscale
**Describe the bug**
Not necessarily a bug but I find it annoying that networkpolicies aren't configured properly when running on exoscale.

On exoscale 
- `networkPolicies.global.externalLoadBalancer` and
- `networkPolicies.global.ingressUsingHostNetwork` should be set to `true`

We configure things such as ingress-nginx on a per cloud basis and so I only think it makes sense that we do the same for the above fields.

**To Reproduce**
Init new environment and don't configure said variables and notice that cert-manager for example can't issue certificates on exsocale.

**Expected behavior**
After init the said variables should have been configured correctly for each cloud.

**Version (add all relevant versions):**
  - Compliant kubernetes apps version 0.30.0

### PR DESCRIPTION
**Pull request by crssnd**
**What this PR does / why we need it**: to set some values at init for exoscale and rook

**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #1597 

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [x] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [x] I upgraded no Chart.
    - [ ] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

### CODE REVIEW

File: migration/v0.31/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+### Prepare upgrade - *non-disruptive*
+
+> *Done before maintenance window.*
+
+1. Pull the latest changes and switch to the correct branch:
+
+    ```bash
+    git pull
+    git switch -d v0.31.x
+    ```
```
Comment (lunkan93) [inline migration/v0.31/README.md:93]: I guess add a manual step to run the `01-move-rook-values.sh` script here too

**Review by lunkan93 — APPROVED** [review]

File: bin/init.bash [hunk 2]
```diff
@@ -204,6 +202,8 @@ set_nginx_config() {
             use_proxy_protocol=false
             use_host_port=true
             service_enabled=false
+            yq4 --inplace '.networkPolicies.global.externalLoadBalancer = true' "${file}"
+            yq4 --inplace '.networkPolicies.global.ingressUsingHostNetwork = true' "${file}"
```
Comment (OlleLarsson) [inline bin/init.bash:206]: Looks a bit out of place writing here.
Would it make sense to take the same approach as the other fields, aka using replace_set_me?

Comment (crssnd) [reply bin/init.bash:206]: [by default](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/common-config.yaml#L749-L750) this values are set to `false`, not sure if replace_set_me will help in this case

Comment (crssnd) [reply bin/init.bash]: done, please check

**Review by OlleLarsson — APPROVED** [review]
lgtm

**Pull request merged**